packaging/plf-tomcat-resources/src/main/resources/bin/setenv.sh

```diff
@@ -128,6 +128,18 @@ if [ "${EXO_CLUSTER}" = "true" ]; then
           EXO_CLUSTER_HOSTS_TCP_1="${EXO_CLUSTER_HOSTS_TCP_1},${exo_cluster_address}[${exo_cluster_tcp1_port}]"
         fi
 
+        if [ "${exo_cluster_name}"  !=  "${EXO_CLUSTER_NODE_NAME}" ]; then
+          if [ -z $EXO_CLUSTER_OORT_CLOUD ]; then
+            EXO_CLUSTER_OORT_CLOUD="${exo_cluster_http_protocol}://${exo_cluster_address}:${exo_cluster_http_port}/cometd/cometd"
+          else
+            EXO_CLUSTER_OORT_CLOUD="${EXO_CLUSTER_OORT_CLOUD},${exo_cluster_http_protocol}://${exo_cluster_address}:${exo_cluster_http_port}/cometd/cometd"
+          fi
+        fi
+
+        if [ -z $EXO_CLUSTER_OORT_CLOUD ]; then
+          EXO_CLUSTER_OORT_CLOUD=""
+        fi
+
```

Comment:
IMO, this block is not required.

Reply:
In fact, for test purpose, when testing cluster mode without multiple nodes (one single node), the variable will not be initialized because` "${exo_cluster_name}"  !=  "${EXO_CLUSTER_NODE_NAME}"` will always be false.

Reply:
@hbenali I moved it outside for loop to make the test before using it

Reply:
OK understood